src/frontends/onnx/frontend/src/utils/tensor_external_data.hpp

```diff
@@ -6,9 +6,14 @@
 
 #include <onnx/onnx_pb.h>
 
+#include "ngraph/runtime/shared_buffer.hpp"
```

Comment:
please avoid use of ngraph

Reply:
In general agree, but we haven't alias/equivalent of shared_buffer in openvino folder/ov namespace